web/chat/chat.js

```diff
@@ -1,71 +1,71 @@
 /**
- * Many customers use PubNub to implement chat into their solution, so they don't have to 
```

Comment:
There are a lot of whitespace changes in this file.  Can these be reverted?  I am more concerned with causing merge conflicts than anything else.